src/lambda_codebase/initial_commit/bootstrap_repository/adf-build/shared/cdk/cdk_constructs/adf_codebuild.py

```diff
@@ -19,7 +19,7 @@
 
 ADF_DEPLOYMENT_REGION = os.environ["AWS_REGION"]
 ADF_DEPLOYMENT_ACCOUNT_ID = os.environ["ACCOUNT_ID"]
-DEFAULT_CODEBUILD_IMAGE = "UBUNTU_14_04_PYTHON_3_7_1"
+DEFAULT_CODEBUILD_IMAGE = "STANDARD_5_0"
```

Comment:
Unfortunately we cannot change the default unless we release a new major version of ADF.
I will rollback this change for now.